controllers/wishlist.controllers.js

```diff
@@ -122,9 +122,49 @@ async function deleteWishlistItem(req, res, next) {
   });
 }
 
+async function sortWishlistItems(req, res, next) {
+  const ownerId = req.user.id;
+  const { field, order } = req.params;
+  const page = +req.params.page || 1;
+  const itemsPerPage = 8;
+
+  const skip = (page - 1) * itemsPerPage;
+
+  // All sort settings
+  let sort = {};
+  sort[field] = order;
+
+  let sortedItems, countItems, pageCount;
+  try {
+    countItems = await WishlistItem.countDocuments({ ownerId });
+
+    sortedItems = await WishlistItem.find({ ownerId })
+      .sort({ ...sort, _id: 1 })
+      .limit(itemsPerPage)
+      .skip(skip);
```

Comment:
Shouldn't skip be placed before limit?
```suggestion
      .skip(skip);
      .limit(itemsPerPage)
```

Reply:
As far as I understand, MongoDB optimizes these queries, so there is no need to change the order of requests

Reply:
Gotcha. I was wondering because I have the same logic but in aggregation, the order must matter there because otherwise it doesn't work. Thanks for clarifying